### DISCUSSION TITLE
part-ii/2021-2/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#118] [Question comments]
Part II 2021 Algebraic Geometry: Paper 3, Section II, I 

https://questions.tripos.org/part-ii/2021-2/

**Comment by eha1234** [comment]
First bits are bookwork. For genus 1 curve, I think intersection with $X_3 = X_4 = 0$ works. For irreducible plane curve of degree 2 isomorphic to $\mathbb{P}_k^1$, either project from a point on $C$, or change coordinates and find an explicit map. The last bits are not bad if you use the fact that a quartic can be written as $x^\top A x$, and we have one or two lines of $A$ is degenerate, a single line if $A$ has rank 1. This also does part (ii): generator is $\det A$ (I think)
